target/linux/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -22,10 +22,17 @@ case "$FIRMWARE" in
 	zyxel,nbg7815)
 		caldata_extract "0:art" 0x1000 0x20000
 		;;
+	prpl,haze)
+                caldata_extract_mmc "0:ART" 0x1000 0x20000
```

Comment:
Where is the caldata stored, on NOR or eMMC?

Reply:
its stored on eMMC